docs/en/reference/Environment-List.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: global
+title: List of Environment Properties
+---
+
+Alluxio supports defining a few frequently used configuration settings through environment
+variables.
+
+<table class="table table-striped">
+<tr><th>Environment Variable</th><th>Description</th></tr>
+{% for env_var in site.data.table.en.env_vars %}
```

Comment:
i don't see this file added to the PR so this would generate an empty table?

Reply:
the table is in docs/_data/table/en/env_vars.yml

Reply:
unless you want me to create a new csv & yml in the generated folder? likewise to config properties tables

Reply:
oh ha didn't realize it was already there. nvm me